workflow/rules/sbs_process.smk

```diff
@@ -0,0 +1,137 @@
+from pathlib import Path
+from lib.shared.file_utils import get_filename
+
+PREPROCESS_FP = ROOT_FP / config["preprocess"]["suffix"]
+SBS_PROCESS_FP = ROOT_FP / config["sbs_process"]["suffix"]
+
+
+# Align images from each sequencing round
+rule align:
+    conda:
+        "../envs/sbs_process.yml"
+    input:
+        lambda wildcards: expand(
+            PREPROCESS_FP
+            / "images"
+            / "sbs"
+            / get_filename(
+                {"well": wildcards.well, "tile": wildcards.tile, "cycle": "{cycle}"},
+                "image",
+                "tiff",
+            ),
+            cycle=SBS_CYCLES,
+        ),
+    output:
+        SBS_PROCESS_FP
+        / "images"
+        / get_filename({"well": "{well}", "tile": "{tile}"}, "aligned", "tiff"),
+    params:
+        method="sbs_mean",
+        upsample_factor=1,
+    script:
+        "../scripts/sbs_process/align_cycles.py"
+
+
+# Apply Laplacian-of-Gaussian filter to all channels
+rule log_filter:
+    conda:
+        "../envs/sbs_process.yml"
+    input:
+        SBS_PROCESS_FP
+        / "images"
+        / get_filename({"well": "{well}", "tile": "{tile}"}, "aligned", "tiff"),
+    output:
+        SBS_PROCESS_FP
+        / "images"
+        / get_filename({"well": "{well}", "tile": "{tile}"}, "log_filtered", "tiff"),
+    params:
```

Comment:
One thing that is missing here is resources. We should discuss this on Tuesday. For now, just pin this as something to delve into

Reply:
Yep, thanks for pinning. We can track our thoughts on this in #7